### DISCUSSION TITLE
The automatic branch creation has failed.

### DISCUSSION THREAD

**Discussion by BlackWhite2000** [GitbookIO/community#398] [3. Git Sync]
I previously reported an issue with repository synchronization failure. 
After conducting tests, it was identified that the synchronization failure was due to the absence of branches in the repository. 
However, in reality, GitBook indicates that branches will be automatically created if they do not exist.

![image](https://github.com/GitbookIO/community/assets/39222378/501bb8fe-67ab-4ad7-822f-80ce277c5ba4)

https://github.com/orgs/GitbookIO/discussions/396

**Comment by taranvohra** [comment]
Hey @BlackWhite2000, can you please provide the `ID` of the space so we can investigate this further?
